cider/tests/nextgen/translator/MockRowToColumn.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2022 Intel Corporation.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#ifndef CIDER_EXEC_NEXTGEN_TRANSLATOR_MOCK_ROWTOCOLUMN_SINK_H
+#define CIDER_EXEC_NEXTGEN_TRANSLATOR_MOCK_ROWTOCOLUMN_SINK_H
+
+#include "exec/nextgen/Context.h"
+#include "exec/nextgen/operators/OpNode.h"
+
+namespace cider::exec::nextgen::operators {
+
+// Mock R2C sink, will be modified After R2CTranslator ready
+class MockRowToColumnTranslator : public Translator {
```

Comment:
Minor hint: instead of creating a ```MockRowToColumnTranslator``` to be removed. We can add a class and mark some TODO as follow ups.

Reply:
Roger,  now i think we can added some TODO here.